templates/chisel/scripts/formal/FPV.tcl

```diff
@@ -0,0 +1,31 @@
+# Make sure you have run the nix build command to build the formal results
```

Comment:
Please add a nix script to run `FPV.tcl`, report should be the result of the derivation.